TestFoundation/TestNSOrderedSet.swift

```diff
@@ -62,6 +61,126 @@ class TestNSOrderedSet : XCTestCase {
             index += 1
         }
     }
+    
+    func test_enumerationUsingBlock() {
```

Comment:
This test was copied from `NSArray` tests for coverage on concurrent enumeration